examples/kops-with-vpc/terraform/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  backend "s3" {
```

Comment:
I'm not sure that this example should includ S3 remote state, since setting up remote state should be up to the user.  If it _does_ include the remote state, then the key should depend on the environment name, the bucket name should be customizable, there should be a step in the Makefile/instructions that creates it.